monkey/common/types/utils/__init__.py

```diff
@@ -0,0 +1 @@
+from .socket_address import socketaddress_from_string
```

Comment:
so the package is `socket_address`, but method name starts with `socketaddress`? We need to pick one, either we separate these words with _ or not

Reply:
This is addressed in [2350](https://github.com/guardicore/monkey/pull/2350)